.github/workflows/ci.yml

```diff
@@ -102,12 +102,52 @@ jobs:
     - name: Run dialyzer
       run: mix dialyzer --no-check
 
+  smoke:
+    name: Start and stop the node
+    needs: compile-native
+    runs-on: ubuntu-22.04
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up Elixir
+      uses: erlef/setup-beam@v1
+      with:
+        version-type: strict
+        version-file: .tool-versions
+    - name: Fetch native libraries
+      id: output-cache
+      uses: actions/cache/restore@v3
+      with:
+        path: priv/native/*
+        key: ${{ runner.os }}-native-${{ hashFiles('native/**') }}
+        fail-on-cache-miss: true
+    - name: Restore dependencies cache
+      uses: actions/cache@v3
+      with:
+        path: deps
+        key: ${{ runner.os }}-mix-${{ hashFiles('**/mix.lock') }}
+        restore-keys: ${{ runner.os }}-mix-
+    - name: Install dependencies
+      run: |
+        sudo apt-get install -y protobuf-compiler
+        echo $(protoc --version)
+        mix escript.install --force hex protobuf
+        protoc --elixir_out=./lib proto/libp2p.proto
+        mix deps.get
+    - name: Set up cargo cache
+      uses: Swatinem/rust-cache@v2
+      with:
+        workspaces: ${{ env.RUST_WORKSPACES }}
+    - name: Compile Elixir (Warnings as errors)
+      run: mix compile --warnings-as-errors
+    - name: Run the node
+      run: mix run -- --checkpoint-sync https://sync-mainnet.beaconcha.in/
```

Comment:
who stops this? what is this testing?

Reply:
It stops automagically. `mix run` starts and then stops the application (unless `--no-halt` is specified)

Reply:
Maybe a comment or something would be nice, this is not clear from the command itself.